contracts/src/warpDiscordBot/namesService/write/registerUser.ts

```diff
@@ -0,0 +1,28 @@
+import { validateInputArgumentPresence, validateString } from '../../../utils';
+import { ContractAction, ContractState, ContractResult, usersPrefix } from '../../types/types';
+
+declare const ContractError;
+
+export const registerUser = async (
+  state: ContractState,
+  { input: { id, address } }: ContractAction
+): Promise<ContractResult> => {
+  validateInputArgumentPresence(id, 'id');
+  validateString(id, 'id');
+  validateInputArgumentPresence(address, 'address');
+  validateString(address, 'address');
+
+  const users = state.users;
+
+  if (Object.keys(users).includes(id)) {
```

Comment:
hym, isn't `users` a simple object? so sth like `if (!users.hasOwnProperty(id))` instead of creating an array with keys on the fly?